README.md

```diff
@@ -68,6 +69,7 @@ After writing this method, our new result should be:
 ```
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
+strip_punctuation has to be called before analyze_mood starts checking if certain words are included, otherwise the punctuation will be included in that check. So I put it inside the parameter I passed to analyze_mood, that way "words" wouldn't get there without passing through the strip_punctuation method.
```

Comment:
This works, and your logic is sound. But is there ever a situation where we would want to analyze_mood on text but not strip_punctuation? I'll argue that there probably isn't, in which case we should call strip_punctuation inside the method, instead of in the method call.